app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb

```diff
@@ -1,4 +1,12 @@
 module ManageIQ::Providers::Openstack::CloudManager::Provision::Cloning
+  def find_destination_in_vmdb(ems_ref)
+    vm_model_class.find_by(:ems_id => source.ext_management_system.id, :ems_ref => ems_ref)
```

Comment:
if we will keep this ,maybe we can just call ```super```

Reply:
Make sense, thanks, will update it.